README.md

```diff
@@ -15,7 +15,7 @@ func main() {
 		Interval:     time.Millisecond * 1000,
 		IteratorType: kcl.IteratorTypeLatest,
 	}
-	k, err := kcl.NewStream(sess, os.Getenv("AWS_KINESIS_STREAM"), s, config)
+	k, err := kcl.NewStream(sess, os.Getenv("AWS_KINESIS_ENDPIONT"), os.Getenv("AWS_KINESIS_STREAM"), s, config)
```

Comment:
Thanks for your contribution, will definitely merge, but can we fix the slight typo?